lib/knapsack_pro/config/env.rb

```diff
@@ -284,6 +284,11 @@ def ci?
             detected_ci != KnapsackPro::Config::CI::Base
         end
 
+        def fallback_mode_error_exit_code_or(default)
+          env_value = ENV['KNAPSACK_PRO_FALLBACK_MODE_ERROR_EXIT_CODE']
```

Comment:
> I would add a unit test in spec/knapsack_pro/config/env_spec.rb for KNAPSACK_PRO_FALLBACK_MODE_ERROR_EXIT_CODE to explain there that 1 means the default exit code when KNAPSACK_PRO_FALLBACK_MODE_ERROR_EXIT_CODE is not defined.

I think it's clear enough by looking at the code, no need to add test cases.

> I would rename fallback_mode_error_exit_code_or to fallback_mode_error_exit_code.
> No need to define 1 as default exit code each time you call this method in multiple test runners. By default 1 is exit code in case of an error.

Great point. Applied.